src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,120 @@
+//navbar
+import { Fragment } from "react";
+import { Disclosure, Menu, Transition } from "@headlessui/react";
+import { Bars3Icon, BellIcon, XMarkIcon } from "@heroicons/react/24/outline";
+
+const navigation = [
+  { name: "Dashboard", href: "#", current: true },
+  { name: "Team", href: "#", current: false },
+  { name: "Projects", href: "#", current: false },
+  { name: "Calendar", href: "#", current: false },
+];
+
+function classNames(...classes) {
+  return classes.filter(Boolean).join(" ");
+}
+
+const Navbar = () => {
```

Comment:
This function appears quite large . It would be better to break it down into smaller components, each with a specific purpose. This approach will significantly improve the readability and maintainability of the code